apps/trading/client-pages/markets/use-markets-stats.tsx

```diff
@@ -88,8 +88,7 @@ export const useTotalVolumeLocked = (
       ?.filter((e) => e?.node?.type === AccountType.ACCOUNT_TYPE_INSURANCE)
       .map((e) => e?.node);
     const balance = accounts?.reduce((acc, a) => {
-      const balance = toQUSD(a?.balance || 0, a?.asset.quantum || 1);
-      return toBigNum(balance || 0, a?.asset.decimals || 0).plus(acc);
+      return toBigNum(a?.balance || 0, a?.asset.decimals || 0).plus(acc);
```

Comment:
I'm not sure that adding balances this way is correct, the $3 + €4 + PLN 1 != 8.
So the previous quantum was ok IMO, but loose the `toBigNum` it's not needed.

```javascript
const balance = toQUSD(a?.balance || 0, a?.asset.quantum || 1);
return balance.plus(acc);
```